examples/tensorflow-mnist-pixel-threshold/demo.ipynb

```diff
@@ -4,14 +4,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Tensorflow MNIST Pixel Threshold demo"
+    "# Tensorflow MNIST Pixel Threshold demo for Dioptra deployment"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This notebook contains a demonstration of how to use Dioptra to run experiments that investigate the effects of the pixel threshold attack when launched on a neural network model trained on the MNIST dataset."
+    "This notebook contains an end-to-end demostration for the Dioptra architecture that is deployed on the DGX workstation."
```

Comment:
This notebook looks like it is based off of https://github.com/usnistgov/dioptra/blob/main/examples/tensorflow-mnist-classifier/demo-dgx-lab-deployment.ipynb, please refer to this comment for more details/guidance: https://github.com/usnistgov/dioptra/pull/12#discussion_r717780817.